src/python/tests/test_parser.py

```diff
@@ -0,0 +1,157 @@
+#
+# This file is part of udmp-parser project
+#
+# Released under MIT License, by 0vercl0k - 2023
+#
+# With contribution from:
+# * hugsy - (github.com/hugsy)
+#
+
+import lief
+import pathlib
+import platform
+import pytest
+import subprocess
+import tempfile
+import time
+import unittest
+
+# format: on
+import udmp_parser  # type: ignore
+from .utils import generate_minidump_from_process_name  # type: ignore
+
+# format: off
+
+TARGET_PROCESS_NAME: str = "winver.exe"
+TARGET_PROCESS_PATH: pathlib.Path = pathlib.Path(
+    f"C:/Windows/System32/{TARGET_PROCESS_NAME}"
+)
+
+
+@pytest.mark.skipif(
+    platform.system().lower() != "windows", reason="Tests only for Windows"
+)
+class TestParserBasic(unittest.TestCase):
+    def setUp(self):
+        # TODO switch to LFS to store minidump test cases (x86, x64, wow64, etc.)
```

Comment:
I'll take care of those in a next PR if you want

Reply:
SGTM!